### DISCUSSION TITLE
Generate a QR Code to your form

### DISCUSSION THREAD

**Discussion by SebastianJurgenson** [team-forms/community#4] [Ideas]
Hi Team!

First of all, I just want to say I love the product!

It would be great if you could generate a QR code that people can scan that will take them directly to there form. This would be accessing the forms a lot easier for our workers on site!

**Comment by SSDK-Hello** [comment]
This is actually already resolved and is under the standard share options on your team form.

![Team Form sharing QR Code](https://i.imgur.com/M6NkEsl.png)

If you need more options you can turn any URL into a QR Code using the below bookmarklet

`javascript:(function(){var%20qrSrc='https://chart.googleapis.com/chart%3Fchs=250x250%26cht=qr%26chl='+encodeURIComponent(document.location.href),overlay=document.createElement('div'),os=overlay.style,img=document.createElement('img');img.src=qrSrc;os.position='fixed';os.zIndex=1000;os.width='100%25';os.height='100%25';os.top=0;os.left=0;os.textAlign='center';os.backgroundColor='rgba(0,0,0,0.9)';img.style.marginTop='100px';overlay.appendChild(img);document.body.appendChild(overlay);overlay.addEventListener('click',function(){document.body.removeChild(overlay);})})();`

If you add any website to your favourites (CTRL+D), then edit the bookmarks address to the above you will have it setup, then just click it.

If you ever need to modify the URL but keep the QR Code you same you will have to use a service such as https://www.the-qrcode-generator.com/
